contracts/src/TokenHome/ERC20TokenHome.sol

```diff
@@ -27,8 +27,25 @@ import {CallUtils} from "../utils/CallUtils.sol";
 contract ERC20TokenHome is IERC20TokenHome, TokenHome {
     using SafeERC20 for IERC20;
 
-    /// @notice The ERC20 token this home contract transfers to TokenRemote instances.
-    IERC20 public token;
+    // solhint-disable private-vars-leading-underscore
+    /// @custom:storage-location erc7201:avalanche-ictt.storage.ERC20TokenHome
+    struct ERC20TokenHomeStorage {
+        /// @notice The ERC20 token this home contract transfers to TokenRemote instances.
+        IERC20 _token;
+    }
+    // solhint-enable private-vars-leading-underscore
+
+    // keccak256(abi.encode(uint256(keccak256("avalanche-ictt.storage.ERC20TokenHome")) - 1)) & ~bytes32(uint256(0xff));
+    bytes32 private constant _ERC20_TOKEN_HOME_STORAGE_LOCATION =
+        0x914a9547f6c3ddce1d5efbd9e687708f0d1d408ce129e8e1a88bce4f40e29500;
+
+    // solhint-disable ordering
+    function _getERC20TokenHomeStorage() private pure returns (ERC20TokenHomeStorage storage $) {
```

Comment:
Is `$` a special token? Or is it just the name of the variable? If the latter, is `$` standard practice for namespaced storage slots?

Reply:
Latter, and yup OZ contracts use it throughout so I decided to follow that convention